llvm/lib/CodeGen/PrologEpilogInserter.cpp

```diff
@@ -1515,11 +1515,6 @@ void PEI::replaceFrameIndicesBackward(MachineBasicBlock *BB,
         break;
     }
 
-    // Refresh the scavenger's internal iterator in case MI was removed or more
```

Comment:
Nice cleanup here. `LocalRS->MBBI` now points after `MI`, to an instruction which will not have been affected by the call to `TRI.eliminateFrameIndex`, so there is no need to refresh anything.